skimage/feature/corner.py

```diff
@@ -67,41 +67,59 @@ def structure_tensor(image, sigma=1, mode='constant', cval=0):
     cval : float, optional
         Used in conjunction with mode 'constant', the value outside
         the image boundaries.
+    order : {'rc', 'xy'}, optional
+        This parameter allows for the use of reverse or forward order of
+        the image axes in gradient computation. 'rc' indicates the use of
+        the first axis initially (Arr, Arc, Acc), whilst 'xy' indicates the
+        usage of the last axis initially (Axx, Axy, Ayy).
 
     Returns
     -------
-    Axx : ndarray
+    Arr : ndarray
```

Comment:
This return value is no longer correct for ndim > 2, is it?

Reply:
Let's leave this one to the follow-up PR adding tests and examples for ndim > 2 support